test/inductor/test_flex_decoding.py

```diff
@@ -856,62 +871,62 @@ def seq_mask_mod(score, b, h, q, kv):
     @supported_platform
     def test_non_divisible_offset_mask(self):
         KV_S = S - 3
-        offset_tensor = torch.tensor(S // 2 - 3, device="cuda", dtype=torch.int32)
+        offset_tensor = torch.tensor(S // 2 - 3, device=GPU_TYPE, dtype=torch.int32)
 
         def mask_mod(b, h, q, kv):
             return kv >= q + offset_tensor
 
-        block_mask = create_block_mask(mask_mod, B, 1, 1, KV_S)
+        block_mask = create_block_mask(mask_mod, B, 1, 1, KV_S, device=self.device)
```

Comment:
May I know why this uses `self.device` but uses `GPU_TYPE` in line 874.

Reply:
This was in the original pull request. I don't think there's a difference and can bring everything to the same form. Which option do you like more: `GPU_TYPE` or `self.device`?

Reply:
Either one is OK Use one code style if possible. IMHO, `self.device` is better.

Reply:
In this file, `GPU_TYPE` and `self.device` are same. I was referencing the way `test_flex_attention.py` is coded.
In that UT file, `self.device` could be set to `cpu`.